sdk/src/main/java/com/mapbox/maps/MapSurface.kt

```diff
@@ -49,6 +50,10 @@ class MapSurface @JvmOverloads constructor(
    */
   fun surfaceCreated() {
     renderer.surfaceCreated()
+    // display should not be null at this point but to be sure we will fallback to DEFAULT_FPS
+    val screenRefreshRate = (context.getSystemService(Context.WINDOW_SERVICE) as WindowManager)
+      .defaultDisplay?.refreshRate?.toInt() ?: MapView.DEFAULT_FPS
+    mapController.setScreenRefreshRate(screenRefreshRate)
```

Comment:
Is there other way to get the refresh rate? the MapSurface is also used for Android Auto, I'm not sure we should always take the default display's refresh rate.

Reply:
Could you please clarify a bit where should we get default screen refresh rate from if not from an Android service? I'm not that familiar with AA setup.
IMO we could always expose (if needed) new method on `MapSurface`, e.g. `MapSurface#setScreenRefreshRate()` to allow control that from outside but for now I doubt doing that as the setup is not fully clear to me